app/views/shared/template/inline-template.html

```diff
@@ -0,0 +1,295 @@
+<html>
+
+  <body>
+    <div style="background-color: #e0f1f3 ;margin: 0;padding: 0;font-family: 'Microsoft YaHei';">
+      <!-- header -->
+      <div id="header" style="margin:0px auto;max-width:600px;">
+        <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;">
+          <tbody>
+            <tr>
+              <td style="direction:ltr;font-size:0px;padding:20px 0;text-align:center;">
+                <div style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;">
+                  <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;" width="100%">
```

Comment:
role跟border也是，省略的話會有影響嗎？

Reply:
tells the screen readers to remove any semantic meaning from the tables—so instead of reading out row and column numbers, it focuses on the content instead.
通常是for 視障者的